flake.nix

```diff
@@ -33,6 +33,7 @@
               # any dev tools you use in excess of the rust ones
               nativeBuildInputs = old.nativeBuildInputs ++ (
                 with pkgs; [
+                  nix
```

Comment:
I think this would only fix the plugin in the context of the shell, so you probably want to also mention this restriction in the README in case the user wants to use it outside the shell (or patch in their own Nix version).  However, this will be good enough for me